### PR TITLE
Zombies? zombies!

### PR DESCRIPTION
**Pull request by kiVts**
# About the pull request
_Oh god its kivts again_
okay so, zombies, lots and lots and lots of things, still wip because Im missing sprites for actions(currently using headbite sprite for both), but it works(UPD, the action buttons not updating, will fix later huh).

what it does
- gives zombies two xeno like actions, bite and leap. leap have 10 second cooldown, probabbly not the only ones in the future, but currently - yes.
- bite infects with 100% chance, 3 second windup, 6 second cooldown
- leap is a slow runner pounce (literally pounce code)
- nerfed zombies bullet resistance and their claws(40>30)
- spaceacillin given a purpose, it can(key word CAN, its just 3 percent chance to heal on level one) now heal stage one zombie virus, also while at it hardens the recipe by a quite a bit and makes the od much much more restrictive
- antibiotic property, same as above but can be created by research(and be made better!).
- removes ""what abouts" information about zombie virus for dead mobs
- removed around 10 if(prob) in black_goo code and replaced it with pick() so its hopefully better
- expands on human abilities a bit to be better for zombies
- removed slash on infect entirely, only bites can infect
- Fixed  1 hour long revive timer

reviving timer?  ~~havent touched that yet~~ (this shit is fixed now, if correct)

# Explain why it's good for the game

someone wanted to remove it so here we are

# Testing Photographs and Procedure
<summary>Screenshots & Videos</summary>

I tried to record gifs but that went horribly for god knows what reason , so **NO GIFS FOR YOU**
</details>

# Changelog

:cl: kivts
add: Two new xeno-like abilities to zombies
add: spaceacillin has a small chance to cure zombie virus
code: Black goo code is now generally better
/:cl:

### CODE REVIEW

**Comment by carlarctg** [comment]
holy shit

**Review by Agameofscones — APPROVED** [review]
sprite approved

**Comment by kiVts** [comment]
hold on but its reusing the same ico- ah well

**Comment by kiVts** [comment]
gonna go into draft because its certainly not over

**Comment by kiVts** [comment]
-removed zombie chance to infect on hit entirely, you can now only infect with bites
-fixed the thing with action buttons update, and it now uses pounce iccon
-spaceacillin now slows down progression of the virus at stage two and stage three. on stage two it halfs the progression rate and on stage three its * 0.75. stage 4 untouched
- adjustments to disease.dm to add the above

**Comment by kiVts** [comment]
on the matter of revive timers being very long, I have no idea whats causing this since on local it works perfectly even with 20 zombs going revival process at the same time. probably something with server!!

**Comment by kiVts** [comment]
you can remotly bite people as zombie. will fix this later

**Comment by kiVts** [comment]
okay, fixed some cooldown jokes I had, also zombies cant harm dead mobs(still can bite though!), also added cooldown messages like in xeno actions. I think Its ready

**Comment by Diegoflores31** [comment]
> okay, fixed some cooldown jokes I had, also zombies cant harm dead mobs(still can bite though!), also added cooldown messages like in xeno actions. I think if nothing comes up its ready

what if abilities had a cost ? like you get powerfull the more people you murder ex. extra range on your leap or less cooldown and maybe bite could be something that is like 10 seconds and goes up to 5 depending on your kills

**Comment by kiVts** [comment]
> > okay, fixed some cooldown jokes I had, also zombies cant harm dead mobs(still can bite though!), also added cooldown messages like in xeno actions. I think if nothing comes up its ready
> 
> what if abilities had a cost ? like you get powerfull the more people you murder ex. extra range on your leap or less cooldown and maybe bite could be something that is like 10 seconds and goes up to 5 depending on your kills

so like abom? That sounds easy.

File: code/datums/disease.dm [hunk 1]
```diff
@@ -42,10 +42,10 @@ var/list/diseases = typesof(/datum/disease) - /datum/disease
 	var/age = 0 // age of the disease in the current mob
 	var/stage_minimum_age = 0 // how old the disease must be to advance per stage
 	var/survive_mob_death = FALSE //whether the virus continues processing as normal when the affected mob is dead.
-
+	var/age_multiplier = 1 //Helpfull to slow down progression of virruses, <1 is slower, >1 is faster. 1 is default
```
Comment (harryob) [inline code/datums/disease.dm:45]:
```suggestion
	/// Helpful to slow down progression of viruses, <1 is slower, >1 is faster. 1 is default
	var/age_multiplier = 1 
```

File: code/datums/diseases/black_goo.dm [hunk 2]
```diff
@@ -23,66 +23,63 @@
...
 		if(1)
-			if(H.stat == DEAD && stage_counter != stage)
-				to_chat(H, SPAN_CENTERBOLD("Your zombie infection is now at Stage One! Zombie transformation begins at Stage Four."))
-				stage_counter = stage
 			survive_mob_death = TRUE //changed because infection rate was REALLY horrible.
 			if(goo_message_cooldown < world.time )
-				if(prob(3))
-					to_chat(affected_mob, SPAN_DANGER("You feel really warm..."))
-					goo_message_cooldown = world.time + 100
+				var/message = pick("Your mouth feels really dry..", "Your head aches...",)
```
Comment (harryob) [inline code/datums/diseases/black_goo.dm:37]:
```suggestion
				var/message = pick("Your mouth feels really dry...", "Your head aches...",)
```

File: code/datums/diseases/black_goo.dm [hunk 3]
```diff
@@ -23,66 +23,63 @@
...
+				if(prob(10))
 					H.vomit_on_floor()
 		if(4)
-			if(H.stat == DEAD && stage_counter != stage)
-				to_chat(H, SPAN_CENTERBOLD("Your zombie infection is now at Stage Four! Your transformation will happen any moment now."))
-				stage_counter = stage
+			H.nutrition = NUTRITION_VERYLOW //brains tasty yey :D
 			H.next_move_slowdown = max(H.next_move_slowdown, 2)
-			if(prob(5) || age >= stage_minimum_age-1)
+			var/message = pick("MAKE IT STOP", "Your skin is pulling itself apart!", "ITS OVER SOON")
```
Comment (harryob) [inline code/datums/diseases/black_goo.dm:67]:
```suggestion
			var/message = pick("MAKE IT STOP!", "Your skin is pulling itself apart!", "IT'S OVER SOON!")
```

File: code/datums/diseases/black_goo.dm [hunk 4]
```diff
@@ -131,26 +137,19 @@
 /obj/item/weapon/zombie_claws/attack(mob/living/target, mob/living/carbon/human/user)
 	if(iszombie(target))
 		return FALSE
-
+	if(target.stat == DEAD)
+		to_chat(user, SPAN_XENOWARNING("[target.name] is already dead, why would you want to do that?"))
```
Comment (harryob) [inline code/datums/diseases/black_goo.dm:141]:
```suggestion
		to_chat(user, SPAN_XENOWARNING("[target] is already dead, why would you want to do that?"))
```

File: code/modules/mob/living/carbon/human/human_abilities.dm [hunk 5]
```diff
@@ -479,6 +491,109 @@ CULT
...
+		return
+	if (!ishuman_strict(M))
+		return
+	if(!action_cooldown_check())
+		return
+	if(get_dist(zombie, M) > 1)
+		return
+	var/mob/living/carbon/human/victim = M
+	var/datum/disease/black_goo/D = locate() in victim.viruses
+	if(D)
```
Comment (harryob) [inline code/modules/mob/living/carbon/human/human_abilities.dm:516]: D -> located_disease

File: code/modules/mob/living/carbon/human/human_abilities.dm [hunk 6]
```diff
@@ -479,6 +491,109 @@ CULT
...
+		to_chat(zombie, SPAN_XENOWARNING("the target is already infected, why would you bite it again?"))
```
Comment (harryob) [inline code/modules/mob/living/carbon/human/human_abilities.dm:517]:
```suggestion
		to_chat(zombie, SPAN_XENOWARNING("The target is already infected, why would you bite it again?"))
```

File: code/modules/mob/living/carbon/human/human_abilities.dm [hunk 7]
```diff
@@ -479,6 +491,109 @@ CULT
...
+	else
+		enter_cooldown(10 SECONDS)
+		to_chat(zombie, SPAN_XENOWARNING("You reach down to [M.name] neck, preparing to bite it!"))
+		to_chat(M, SPAN_DANGER("[zombie.name] reaches to your neck and starts to open his jaw, agh!"))
+		if(do_after(zombie, 5 SECONDS, INTERRUPT_ALL, BUSY_ICON_HOSTILE, M, INTERRUPT_MOVED, BUSY_ICON_HOSTILE))
```
Comment (harryob) [inline code/modules/mob/living/carbon/human/human_abilities.dm:522]: early return

ie
```dm
if(!do_after)
 to_chat(zombie, "interrupted")
  return

to_chat(zombie, SPAN_XENOWARNING...
```

File: code/modules/mob/living/carbon/human/human_abilities.dm [hunk 8]
```diff
@@ -479,6 +491,109 @@ CULT
...
+		if (!do_after(zomb, windup_duration, INTERRUPT_NO_NEEDHAND, BUSY_ICON_HOSTILE))
+			to_chat(zomb, SPAN_XENODANGER("You cancel your [name]!"))
+			return
+
+		if (!windup_interruptable)
+			zomb.frozen = FALSE
+			zomb.anchored = FALSE
+			zomb.update_canmove()
+
+	zomb.visible_message(SPAN_XENOWARNING("\The [zomb.name] [name][findtext(name, "e", -1) || findtext(name, "p", -1) ? "s" : "es"] at [target]!"), SPAN_XENOWARNING("You [name] at [target]!"))
```
Comment (harryob) [inline code/modules/mob/living/carbon/human/human_abilities.dm:584]:
```suggestion
	zomb.visible_message(SPAN_XENOWARNING("[zomb] [name][findtext(name, "e", -1) || findtext(name, "p", -1) ? "s" : "es"] at [target]!"), SPAN_XENOWARNING("You [name] at [target]!"))
```

File: code/modules/mob/living/carbon/human/human_abilities.dm [hunk 9]
```diff
@@ -479,6 +491,109 @@ CULT
...
+	if(D)
+		to_chat(zombie, SPAN_XENOWARNING("the target is already infected, why would you bite it again?"))
+	else
+		enter_cooldown(10 SECONDS)
+		to_chat(zombie, SPAN_XENOWARNING("You reach down to [M.name] neck, preparing to bite it!"))
+		to_chat(M, SPAN_DANGER("[zombie.name] reaches to your neck and starts to open his jaw, agh!"))
+		if(do_after(zombie, 5 SECONDS, INTERRUPT_ALL, BUSY_ICON_HOSTILE, M, INTERRUPT_MOVED, BUSY_ICON_HOSTILE))
+			to_chat(zombie, SPAN_XENOWARNING("You bite [M.name] neck, leaving a bloody mark!"))
+			to_chat(M, SPAN_DANGER("[zombie.name] Bites you right in the neck!"))
+			playsound(zombie.loc, 'sound/hallucinations/wail.ogg', 25, TRUE)
```
Comment (harryob) [inline code/modules/mob/living/carbon/human/human_abilities.dm:525]:
```suggestion
			playsound(get_turf(zombie), 'sound/hallucinations/wail.ogg', 25, TRUE)
```
why not just play sound from the zombie though

File: code/modules/mob/living/carbon/human/human_abilities.dm [hunk 10]
```diff
@@ -479,6 +491,109 @@ CULT
...
+	var/datum/disease/black_goo/D = locate() in victim.viruses
+	if(D)
+		to_chat(zombie, SPAN_XENOWARNING("the target is already infected, why would you bite it again?"))
+	else
+		enter_cooldown(10 SECONDS)
+		to_chat(zombie, SPAN_XENOWARNING("You reach down to [M.name] neck, preparing to bite it!"))
+		to_chat(M, SPAN_DANGER("[zombie.name] reaches to your neck and starts to open his jaw, agh!"))
+		if(do_after(zombie, 5 SECONDS, INTERRUPT_ALL, BUSY_ICON_HOSTILE, M, INTERRUPT_MOVED, BUSY_ICON_HOSTILE))
+			to_chat(zombie, SPAN_XENOWARNING("You bite [M.name] neck, leaving a bloody mark!"))
+			to_chat(M, SPAN_DANGER("[zombie.name] Bites you right in the neck!"))
```
Comment (harryob) [inline code/modules/mob/living/carbon/human/human_abilities.dm:524]:
```suggestion
			to_chat(M, SPAN_DANGER("[zombie] bites you right in the neck!"))
```

File: code/modules/mob/living/carbon/human/human_abilities.dm [hunk 11]
```diff
@@ -479,6 +491,109 @@ CULT
...
+	var/mob/living/carbon/human/victim = M
+	var/datum/disease/black_goo/D = locate() in victim.viruses
+	if(D)
+		to_chat(zombie, SPAN_XENOWARNING("the target is already infected, why would you bite it again?"))
+	else
+		enter_cooldown(10 SECONDS)
+		to_chat(zombie, SPAN_XENOWARNING("You reach down to [M.name] neck, preparing to bite it!"))
+		to_chat(M, SPAN_DANGER("[zombie.name] reaches to your neck and starts to open his jaw, agh!"))
+		if(do_after(zombie, 5 SECONDS, INTERRUPT_ALL, BUSY_ICON_HOSTILE, M, INTERRUPT_MOVED, BUSY_ICON_HOSTILE))
+			to_chat(zombie, SPAN_XENOWARNING("You bite [M.name] neck, leaving a bloody mark!"))
```
Comment (harryob) [inline code/modules/mob/living/carbon/human/human_abilities.dm:523]:
```suggestion
			to_chat(zombie, SPAN_XENOWARNING("You bite [M] neck, leaving a bloody mark!"))
```

File: code/modules/mob/living/carbon/human/human_abilities.dm [hunk 12]
```diff
@@ -479,6 +491,109 @@ CULT
...
+	if(get_dist(zombie, M) > 1)
+		return
+	var/mob/living/carbon/human/victim = M
+	var/datum/disease/black_goo/D = locate() in victim.viruses
+	if(D)
+		to_chat(zombie, SPAN_XENOWARNING("the target is already infected, why would you bite it again?"))
+	else
+		enter_cooldown(10 SECONDS)
+		to_chat(zombie, SPAN_XENOWARNING("You reach down to [M.name] neck, preparing to bite it!"))
+		to_chat(M, SPAN_DANGER("[zombie.name] reaches to your neck and starts to open his jaw, agh!"))
```
Comment (harryob) [inline code/modules/mob/living/carbon/human/human_abilities.dm:521]:
```suggestion
		to_chat(M, SPAN_DANGER("[zombie.name] reaches your neck and starts to open [zombie.p_their()] jaw!"))
```

File: code/modules/mob/living/carbon/human/human_abilities.dm [hunk 13]
```diff
@@ -479,6 +491,109 @@ CULT
...
+		return
+	if(get_dist(zombie, M) > 1)
+		return
+	var/mob/living/carbon/human/victim = M
+	var/datum/disease/black_goo/D = locate() in victim.viruses
+	if(D)
+		to_chat(zombie, SPAN_XENOWARNING("the target is already infected, why would you bite it again?"))
+	else
+		enter_cooldown(10 SECONDS)
+		to_chat(zombie, SPAN_XENOWARNING("You reach down to [M.name] neck, preparing to bite it!"))
```
Comment (harryob) [inline code/modules/mob/living/carbon/human/human_abilities.dm:520]:
```suggestion
		to_chat(zombie, SPAN_XENOWARNING("You reach down to [M] neck, preparing to bite it!"))
```

File: code/modules/mob/living/carbon/human/human_abilities.dm [hunk 14]
```diff
@@ -170,7 +172,17 @@ CULT
...
+	addtimer(CALLBACK(src, PROC_REF(on_cooldown_over)), amount, TIMER_UNIQUE|TIMER_STOPPABLE)
+
+/datum/action/human_action/activable/proc/on_cooldown_over()
+	if(!owner)
+		return
+	update_button_icon()
+	if(cooldown_message_on)
+		if(isnull(cooldown_message))
+			to_chat(owner, SPAN_XENODANGER("You feel your strength return! You can use [name] again!"))
+		else
```
Comment (harryob) [inline code/modules/mob/living/carbon/human/human_abilities.dm:184]: i don't quite get why this is like this, if it's just copypasta let me know but uhhh... we have TIMER_COOLDOWN_START and stuff

File: code/modules/mob/living/carbon/human/species/zombie.dm [hunk 15]
```diff
@@ -24,9 +24,12 @@
 	stun_reduction = 10
 	knock_out_reduction = 5
 	has_organ = list()
-
 	has_species_tab_items = TRUE
-
+	var/d
```
Comment (harryob) [inline code/modules/mob/living/carbon/human/species/zombie.dm:28]: erm?

File: code/modules/reagents/chemistry_properties/prop_positive.dm [hunk 16]
```diff
@@ -900,6 +900,35 @@
...
+	value = 1
+	max_level = 3
+
+/datum/chem_property/positive/antibiotic/process(mob/living/M, potency = 1, delta_time)
+	M.apply_damage(0.5 * potency * delta_time, TOX)
+	var/datum/disease/black_goo/D = locate() in M.viruses
+	if(D)
+		var/datum/disease/black_goo/virus = new
+		var/mob/living/carbon/human/H = M
+		switch(virus.stage)
```
Comment (harryob) [inline code/modules/reagents/chemistry_properties/prop_positive.dm:918]: does this need to be a switch if there's only one condition

File: code/modules/reagents/chemistry_properties/prop_positive.dm [hunk 17]
```diff
@@ -900,6 +900,35 @@
...
+	category = PROPERTY_TYPE_MEDICINE
+	value = 1
+	max_level = 3
+
+/datum/chem_property/positive/antibiotic/process(mob/living/M, potency = 1, delta_time)
+	M.apply_damage(0.5 * potency * delta_time, TOX)
+	var/datum/disease/black_goo/D = locate() in M.viruses
+	if(D)
+		var/datum/disease/black_goo/virus = new
+		var/mob/living/carbon/human/H = M
```
Comment (harryob) [inline code/modules/reagents/chemistry_properties/prop_positive.dm:917]: no one letter vars...

File: code/modules/reagents/chemistry_properties/prop_positive.dm [hunk 18]
```diff
@@ -900,6 +900,35 @@
...
+	code = "ABT"
+	description = "Blocks certain biological processes from occuring in all living tissue, while it helps with infections, it also harms the body itself. overdosis can cause extreme lethality"
+	rarity = PROPERTY_COMMON
+	category = PROPERTY_TYPE_MEDICINE
+	value = 1
+	max_level = 3
+
+/datum/chem_property/positive/antibiotic/process(mob/living/M, potency = 1, delta_time)
+	M.apply_damage(0.5 * potency * delta_time, TOX)
+	var/datum/disease/black_goo/D = locate() in M.viruses
```
Comment (harryob) [inline code/modules/reagents/chemistry_properties/prop_positive.dm:914]: no one letter vars...

**Review by harryob — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/species/zombie.dm (hunk 15 above)
Comment (kiVts) [reply code/modules/mob/living/carbon/human/species/zombie.dm:28]: I have no idea either

File: code/modules/mob/living/carbon/human/human_abilities.dm (hunk 14 above)
Comment (kiVts) [reply code/modules/mob/living/carbon/human/human_abilities.dm:184]: I really just looked at xeno actions and looked at human_abilities... yeha

File: code/modules/mob/living/carbon/human/human_abilities.dm (hunk 5 above)
Comment (kiVts) [reply code/modules/mob/living/carbon/human/human_abilities.dm]: done

File: code/datums/diseases/black_goo.dm [hunk 19]
```diff
@@ -23,66 +23,63 @@
...
-				stage_counter = stage
-			hidden = list(0,0)
-			//survive_mob_death = TRUE //even if host dies now, the transformation will occur.
+			if(victim_has_antibiotic())
+				age_multiplier = 0.75
+			else
+				age_multiplier = 1
+			hidden = list(1,0)
 			H.next_move_slowdown = max(H.next_move_slowdown, 1)
+			var/message = pick("Your muscles ache", "Your throat is really dry", "Skin on your hands peels away.", "You can feel your heart skipping a beat...")
```
Comment (harryob) [inline code/datums/diseases/black_goo.dm:58]:
```suggestion
			var/message = pick("Your muscles ache.", "Your throat is really dry.", "Skin on your hands peels away.", "You can feel your heart skipping a beat...")
```

File: code/datums/diseases/black_goo.dm [hunk 20]
```diff
@@ -94,10 +91,18 @@
 						H.apply_damage(-healamt, OXY)
 				H.nutrition = NUTRITION_MAX //never hungry
 
+/datum/disease/black_goo/proc/victim_has_antibiotic()
+	for(var/i=1 to affected_mob.reagents.reagent_list.len)
+		var/datum/reagent/S = affected_mob.reagents.reagent_list[i]
+		if(S.get_property(PROPERTY_ANTIBIOTIC))
+			return TRUE
+		else
+			return FALSE
```
Comment (harryob) [inline code/datums/diseases/black_goo.dm:100]:
```suggestion
	for(var/datum/reagent/current_reagent as anything in affected_mob.reagents.reagent_list)
		if(current_reagent.get_property(PROPERTY_ANTIBIOTIC))
			return TRUE
	return FALSE
```

File: code/modules/mob/living/carbon/human/human_abilities.dm [hunk 21]
```diff
@@ -479,6 +491,109 @@ CULT
...
+	if(located_disease)
+		to_chat(zombie, SPAN_XENOWARNING("The target is already infected, why would you bite it again?"))
+	else
+		enter_cooldown(10 SECONDS)
+		to_chat(zombie, SPAN_XENOWARNING("You reach down to [M] neck, preparing to bite it!"))
+		to_chat(M, SPAN_DANGER("[zombie.name] reaches your neck and starts to open [zombie.p_their()] jaw!"))
+		if(!do_after(zombie, 5 SECONDS, INTERRUPT_ALL, BUSY_ICON_HOSTILE, M, INTERRUPT_MOVED, BUSY_ICON_HOSTILE))
+			to_chat(zombie, SPAN_XENOWARNING("You were interupted!"))
+			return
+		to_chat(zombie, SPAN_XENOWARNING("You bite [M] neck, leaving a bloody mark!"))
```
Comment (harryob) [inline code/modules/mob/living/carbon/human/human_abilities.dm:525]:
```suggestion
		to_chat(zombie, SPAN_XENOWARNING("You bite [M]'s neck, leaving a bloody mark!"))
```

File: code/modules/mob/living/carbon/human/human_abilities.dm [hunk 22]
```diff
@@ -479,6 +491,109 @@ CULT
...
+		victim.emote("scream")
+		playsound(victim, "bone_break" , 45, TRUE)
+		victim.AddDisease(new /datum/disease/black_goo)
+		zombie.flick_attack_overlay(victim, "bite")
+		zombie.animation_attack_on(victim)
+		victim.apply_damage(20, BRUTE, "head")
+
+
+/datum/action/human_action/activable/zombie/leap
+	name = "leap"
```
Comment (harryob) [inline code/modules/mob/living/carbon/human/human_abilities.dm:537]:
```suggestion
	name = "Leap"
```

File: code/modules/mob/living/carbon/human/human_abilities.dm [hunk 23]
```diff
@@ -479,6 +491,109 @@ CULT
...
+	action_icon_state = "leap"
+	var/maxdistance = 5 //leap how far again?
+	var/throw_speed = SPEED_AVERAGE
+	var/windup = FALSE // Is there a do_after before we pounce?
+	var/windup_duration = 20 // How long to wind up, if applicable
```
Comment (harryob) [inline code/modules/mob/living/carbon/human/human_abilities.dm:542]: dm doc 

so do
/// How far does this leap go?

File: code/modules/mob/living/carbon/human/human_abilities.dm [hunk 24]
```diff
@@ -607,3 +722,6 @@ CULT
 
 	var/mob/living/carbon/human/human_user = owner
 	SEND_SIGNAL(human_user, COMSIG_MOB_MG_EXIT)
+
+
+
```
Comment (harryob) [inline code/modules/mob/living/carbon/human/human_abilities.dm:727]:
```suggestion
```

**Review by harryob — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/human_abilities.dm [hunk 25]
```diff
@@ -479,6 +491,114 @@ CULT
...
+			return
+		to_chat(zombie, SPAN_XENOWARNING("You bite [M]'s neck, leaving a bloody mark!"))
+		to_chat(M, SPAN_DANGER("[zombie] bites you right in the neck!"))
+		playsound(get_turf(zombie), 'sound/hallucinations/wail.ogg', 25, TRUE)
+		victim.emote("scream")
+		playsound(victim, "bone_break" , 45, TRUE)
+		victim.AddDisease(new /datum/disease/black_goo)
+		zombie.flick_attack_overlay(victim, "bite")
+		zombie.animation_attack_on(victim)
+		victim.apply_damage(20, BRUTE, "head")
```
Comment (harryob) [inline code/modules/mob/living/carbon/human/human_abilities.dm:533]:
```suggestion
	if(located_disease)
		to_chat(zombie, SPAN_XENOWARNING("The target is already infected, why would you bite it again?"))
		return

	enter_cooldown(10 SECONDS)
	to_chat(zombie, SPAN_XENOWARNING("You reach down to [M] neck, preparing to bite it!"))
	to_chat(M, SPAN_DANGER("[zombie.name] reaches your neck and starts to open [zombie.p_their()] jaw!"))
	
	if(!do_after(zombie, 5 SECONDS, INTERRUPT_ALL, BUSY_ICON_HOSTILE, M, INTERRUPT_MOVED, BUSY_ICON_HOSTILE))
		to_chat(zombie, SPAN_XENOWARNING("You were interupted!"))
		return
	
	to_chat(zombie, SPAN_XENOWARNING("You bite [M]'s neck, leaving a bloody mark!"))
	to_chat(M, SPAN_DANGER("[zombie] bites you right in the neck!"))
	playsound(get_turf(zombie), 'sound/hallucinations/wail.ogg', 25, TRUE)
	victim.emote("scream")
	playsound(victim, "bone_break" , 45, TRUE)
	victim.AddDisease(new /datum/disease/black_goo)
	zombie.flick_attack_overlay(victim, "bite")
	zombie.animation_attack_on(victim)
	victim.apply_damage(20, BRUTE, "head")
```

File: code/modules/reagents/chemistry_properties/prop_positive.dm [hunk 26]
```diff
@@ -900,6 +900,34 @@
...
+	if(located_disease)
+		var/datum/disease/black_goo/virus = new
+		var/mob/living/carbon/human/infected = M
+		if(virus.stage == 1)
+			if(prob(3 * potency))
+				for(var/datum/disease/goo in infected.viruses)
+					goo.cure()
+			else
+				if(prob(15 * potency))
+					to_chat(M, SPAN_WARNING("Your insides feel insanely hot!"))
```
Comment (harryob) [inline code/modules/reagents/chemistry_properties/prop_positive.dm:924]:
```suggestion
	var/datum/disease/black_goo/located_disease = locate() in M.viruses
	if(!located_disease)
		return
		
	if(located_disease.stage != 1)
		return
	
	if(prob(3 * potency))
		for(var/datum/disease/black_goo/goo in M.viruses)
			goo.cure()
		return

	if(prob(15 * potency))
		to_chat(M, SPAN_WARNING("Your insides feel insanely hot!"))
```

File: code/modules/reagents/chemistry_properties/prop_positive.dm [hunk 27]
```diff
@@ -900,6 +900,34 @@
...
+
+/datum/chem_property/positive/antibiotic/process_overdose(mob/living/M, potency, delta_time)
+	M.apply_damage(4 * potency * delta_time, TOX)
+
+/datum/chem_property/positive/antibiotic/process_critical(mob/living/M, potency, delta_time)
+	M.apply_damage(8 * potency * delta_time, TOX)
```
Comment (harryob) [inline code/modules/reagents/chemistry_properties/prop_positive.dm:930]:
```suggestion
	M.apply_damage(8 * potency * delta_time, TOX)

```

**Review by harryob — CHANGES_REQUESTED** [review]

**Comment by kiVts** [comment]
commits above fix dead zombies making things they shouldnt. 
still on the hunt for the one hour revive timer

**Comment by kiVts** [comment]
this fixes one hour revive timer

**Comment by kiVts** [comment]
image

I'm just going to kill this sooner than later by myself because that's what they meant. For whatever reason PR that makes zombies better after someone attempts to remove them is not a good enough reason. (^morrow said the text in screenshot btw) + Im tired of this dumb shit. 

Sooo.. Enjoy your sweet zombies?

**Comment by SarahJohnson489** [comment]
We just got the 1 hr revive bug in the last zombies round. If you could do an atomic PR for that it'd be great.

**Comment by kiVts** [comment]
It involves removing unique and override flags from zombie revive timer, No I'm not doing it.

**Pull request closed without merging**